src/app/lib/watch-height.ts

```diff
@@ -0,0 +1,10 @@
+export const watchHeight = () => {
+  const setHeight = () => {
+    const vh = window.innerHeight * 0.01
+
+    document.documentElement.style.setProperty('--vh', `${vh.toFixed(2)}px`)
```

Comment:
Эксперимент показал, что если не умножать на 0.01, а делить на 100, то этого бага не будет. Так что предлагаю сделать вот так:

```suggestion
    const vh = window.innerHeight / 100

    document.documentElement.style.setProperty('--vh', `${vh}px`)
```